python/README.md

```diff
@@ -13,7 +13,7 @@ Updating the list of known versions might be tricky, here are the steps:
 3. Update the list of known Python versions in [`language_version_manager.rb`](https://github.com/dependabot/dependabot-core/blob/main/python/lib/dependabot/python/language_version_manager.rb).
 4. Fix any broken tests.
 
-[Example PR](https://github.com/dependabot/dependabot-core/pull/7412) that does all these things.
+[Example PR](https://github.com/dependabot/dependabot-core/pull/7914) that does all these things.
```

Comment:
👍 for this great attention to detail, with the caveat that this example PR only does a handful of these things. I'm looking forward to when Python 3.12 drops in October and then we can have a single PR showing how to do a sorta major (at least for python) version bump.